src/makeTxn.ts

```diff
@@ -202,7 +202,7 @@ export function makeKeyRegistrationTxnWithSuggestedParamsFromObject(
       >,
       'from' | 'note' | 'suggestedParams' | 'rekeyTo'
     > & {
-      nonParticipation: true;
+      nonParticipation: boolean;
     }
```

Comment:
You can simplify it to this:

```js
  o: Expand<
    Pick<
      RenameProperty<
        MustHaveSuggestedParams<KeyRegistrationTxn>,
        'reKeyTo',
        'rekeyTo'
      >,
      'from' | 'note' | 'suggestedParams' | 'rekeyTo' | 'nonParticipation'
    >
  >
```

Note this indicates a _slightly_ different meaning, since if `nonParticipation` is not included, it defaults to `false`, whereas in your original change it would be a required parameter

Reply:
makes sense,  thanks!